src/Symfony/Component/Validator/Constraints/BicValidator.php

```diff
@@ -63,7 +69,11 @@ public function validate($value, Constraint $constraint)
         }
 
         // next 2 letters must be alphabetic (country code)
-        if (!ctype_alpha(substr($canonicalize, 4, 2))) {
+        if (!class_exists(Intl::class)) {
```

Comment:
IMO this is a BC break. The constraint was working without the Intl component before.

Reply:
and this exception will only be thrown at runtime if no test covers its usage meaning that it will result in a server error which is not good from my point of view

Reply:
I agree. Reading the issue, I thought this was accepted as a bugfix. 
I'm going to submit a BC layer